services/src/handlers/wfs.rs

```diff
@@ -36,133 +40,138 @@ where
-///   <ows:ServiceContact>
-///     <ows:ContactInfo>
-///       <ows:Address>
-///         <ows:ElectronicMailAddress>info@geoengine.de</ows:ElectronicMailAddress>
-///       </ows:Address>
-///     </ows:ContactInfo>
-///   </ows:ServiceContact>
-/// </ows:ServiceProvider>
-/// <ows:OperationsMetadata>
-///   <ows:Operation name="GetCapabilities">
-///     <ows:DCP>
-///       <ows:HTTP>
-///         <ows:Get xlink:href="http://localhost/wfs/93d6785e-5eea-4e0e-8074-e7f78733d988"/>
-///         <ows:Post xlink:href="http://localhost/wfs/93d6785e-5eea-4e0e-8074-e7f78733d988"/>
-///       </ows:HTTP>
-///     </ows:DCP>
-///     <ows:Parameter name="AcceptVersions">
-///       <ows:AllowedValues>
-///         <ows:Value>2.0.0</ows:Value>
-///       </ows:AllowedValues>
-///     </ows:Parameter>
-///     <ows:Parameter name="AcceptFormats">
-///       <ows:AllowedValues>
-///         <ows:Value>text/xml</ows:Value>
-///       </ows:AllowedValues>
-///     </ows:Parameter>
-///   </ows:Operation>
-///   <ows:Operation name="GetFeature">
-///     <ows:DCP>
-///       <ows:HTTP>
-///         <ows:Get xlink:href="http://localhost/wfs/93d6785e-5eea-4e0e-8074-e7f78733d988"/>
-///         <ows:Post xlink:href="http://localhost/wfs/93d6785e-5eea-4e0e-8074-e7f78733d988"/>
-///       </ows:HTTP>
-///     </ows:DCP>
-///     <ows:Parameter name="resultType">
-///       <ows:AllowedValues>
-///         <ows:Value>results</ows:Value>
-///         <ows:Value>hits</ows:Value>
-///       </ows:AllowedValues>
-///     </ows:Parameter>
-///     <ows:Parameter name="outputFormat">
-///       <ows:AllowedValues>
-///         <ows:Value>application/json</ows:Value>
-///         <ows:Value>json</ows:Value>
-///       </ows:AllowedValues>
-///     </ows:Parameter>
-///     <ows:Constraint name="PagingIsTransactionSafe">
-///       <ows:NoValues/>
-///       <ows:DefaultValue>FALSE</ows:DefaultValue>
-///     </ows:Constraint>
-///   </ows:Operation>
-///   <ows:Constraint name="ImplementsBasicWFS">
-///     <ows:NoValues/>
-///     <ows:DefaultValue>TRUE</ows:DefaultValue>
-///   </ows:Constraint>
-/// </ows:OperationsMetadata>
-/// <FeatureTypeList>
-///   <FeatureType>
-///     <Name>93d6785e-5eea-4e0e-8074-e7f78733d988</Name>
-///     <Title>Wofklow 93d6785e-5eea-4e0e-8074-e7f78733d988</Title>
-///     <DefaultCRS>urn:ogc:def:crs:EPSG::4326</DefaultCRS>
-///     <ows:WGS84BoundingBox>
-///       <ows:LowerCorner>-90 -180</ows:LowerCorner>
-///       <ows:UpperCorner>90 180</ows:UpperCorner>
-///     </ows:WGS84BoundingBox>
-///   </FeatureType>
-/// </FeatureTypeList>
-/// </wfs:WFS_Capabilities>
-/// ```
+/// Get WFS Capabilities
+#[utoipa::path(
+    tag = "OGC WFS",
+    get,
+    path = "/wfs/{workflow}?request=GetCapabilities",
+    responses(
+        (status = 200, description = "OK", content_type = "text/xml", body = String,
+            // TODO: add example when utoipa supports more than just json examples
```

Comment:
Is there an issue to refer to?

Or maybe output it as a JSON text blob?

Reply:
There is no GitHub issue. It's more of a missing feature I guess. We could output it as JSON text, but then the content type cannot be text/xml. We can either have the correct type or an example.

Reply:
What about https://github.com/juhaku/utoipa/pull/41 ?

Reply:
I don't exactly know what this PR ist about, all I know is that the `example` parameter of the utoipa path macro only supports Json Values.